### PR TITLE
Minor doc change for GAing Pod DNS Config

### PR DESCRIPTION
**Pull request by MrHohn**
Ref https://github.com/kubernetes/enhancements/issues/504, Pod DNS Config (configurable pod resolv.conf) is becoming GA in k8s 1.14.

### CODE REVIEW

File: content/en/docs/concepts/services-networking/dns-pod-service.md [hunk 1]
```diff
@@ -257,6 +251,22 @@ search default.svc.cluster.local svc.cluster.local cluster.local
...
+
+The availability of Pod DNS Config and DNS Policy "`None`"" is shown as below.
+
+| k8s version | Feature support |
+| :---------: |:-----------:|
+| 1.14 | Stable |
+| 1.10 | Beta (on by default)|
+| 1.9 | Alpha |
+
+To enable this feature in v1.9, the cluster administrator
```
Comment (thockin) [inline content/en/docs/concepts/services-networking/dns-pod-service.md]: Since 1.9 is already out of window, do we need to include text like this?

Comment (MrHohn) [reply content/en/docs/concepts/services-networking/dns-pod-service.md]: Good point, given we are at 1.14 already, I will remove these text and only mention it went alpha in 1.9.

Comment (MrHohn) [reply content/en/docs/concepts/services-networking/dns-pod-service.md]: Updated.

**Pull request merged**